feature-examples/wan/scripts/start-ln.gfsh

```diff
@@ -18,20 +18,27 @@
 #
 
 # Start the locator
-start locator --name=locator-ln --port=10332 --locators=localhost[10332] --mcast-port=0 --log-level=config \
+start locator --name=locator-ln1 --port=10332 --locators=localhost[10332],localhost[10333] --mcast-port=0 --log-level=config \
+        --http-service-port=7070 \
 	--J=-Dgemfire.remote-locators=localhost[10331] --J=-Dgemfire.distributed-system-id=2 \
 	--J=-Dgemfire.jmx-manager-start=true --J=-Dgemfire.jmx-manager-http-port=8082 \
 	--J=-Dgemfire.jmx-manager-port=1092
 
+start locator --name=locator-ln2 --port=10333 --locators=localhost[10333],localhost[10332] --mcast-port=0 --log-level=config \
```

Comment:
Starting multiple locators for the ln cluster is a good idea but since we run the example after connecting to 10332 we won't ever run a command on the second locator right? Do we want to add commands that run on the second locator or swap some of the commands we're running to run on the second locator?

Reply:
I leave it in case to extend for future.